arbitrum-docs/node-running/running-a-node.mdx

```diff
@@ -4,7 +4,7 @@ Note: If you’re interested in accessing an Arbitrum chain, but you don’t wan
 
 ### Required Artifacts
 
-- Latest Docker Image: `offchainlabs/nitro-node:v2.0.9-51f9452`
+- Latest Docker Image: `offchainlabs/nitro-node:v2.0.10-73224e3`
```

Comment:
Need to update the docker image references in the `docker run` command lines below as well

Reply:
Thanks Josh! Changed them!